src/evaluations/tests/analyzer_test.py

```diff
@@ -138,6 +139,32 @@ def test_get_num_estimable_sets_df_results_correct(self):
     except AssertionError:
       self.fail('The number of estimable sets is not correct.')
 
+  def test_get_relative_error_stats_of_num_of_estimable_sets(self):
```

Comment:
Line 58 and 64 are broken by changes set_generator.py changes. I think get_generator_factory_with_num_and_size is intended instead of get_generator_factory but I'm not 100% sure.

Reply:
i think this has been addressed, let me know